docs/create-packages/Package-authoring-best-practices.md

```diff
@@ -60,11 +60,11 @@ Below is a table mapping and describing available package metadata elements:
 | [`Authors`](#authors)                         	| [`Authors`](/nuget/reference/msbuild-targets#pack-target)                                                                	| [`authors`](/nuget/reference/nuspec#authors)                            	| A comma-separated list of package authors, often using the individual's or an organization's "pretty name."       	|
 | [`Description`](#description)                 	| [`Description`](/nuget/reference/msbuild-targets#pack-target)                                                        	| [`description`](/nuget/reference/nuspec#description)                    	| A description of the package.                                                                                     	|
 | [`Copyright`](#copyright)                     	| [`Copyright`](/nuget/reference/msbuild-targets#pack-target)                                                            	| [`copyright`](/nuget/reference/nuspec#copyright)                        	| Copyright details for the package.                                                                                	|
-| [`Project URL`](#project-url)                 	| `PackageProjectUrl`                                                                                                                     	| [`projectUrl`](/nuget/reference/nuspec#projecturl)                      	| A URL for the project homepage.                                                                                   	|
+| [`Project URL`](#project-url)                 	| [`PackageProjectUrl`](/nuget/reference/msbuild-targets#pack-target)                                                       |      [`projectUrl`](/nuget/reference/nuspec#projecturl)                      	| A URL for the project homepage.                                                                                   	|
```

Comment:
Is this URL change intentional?

Reply:
The change here is to make PackageProjectUrl into a link (before it was just text).